### DISCUSSION TITLE
How to publish my game for Xbox?

### DISCUSSION THREAD

**Discussion by paulocoutinhox** [axmolengine/axmol#1765] [General]
Hi,

How to publish my game for Xbox?

What i need add/create/update in my project to add Xbox support?

What are the steps from Axmol side?

Thanks.

**Comment by halx99** [comment]
Generate vs uwp build solution: `axmol build -p winuwp -c`

**Reply by halx99** [reply]
I haven't publish axmol games to xbox, the axmol engine just provide build for winuwp target.

**Comment by aismann** [comment]
@paulocoutinhox 
Please try this description:
https://learn.microsoft.com/en-us/windows/uwp/xbox-apps/getting-started
